### DISCUSSION TITLE
Add FQDN to no_proxy

### DISCUSSION THREAD

**Discussion by EddieN17** [Security-Onion-Solutions/securityonion#11020] [2.4]
Currently the set_proxy function in setup/so-functions doesn't include the FQDN. This could cause an error for people who setups the proxy and uses the FQDN for the web interface during install. As the Elastic Agent install on manager will try enrolling by going through the proxy to reach the URL and fail (at least for me) https://github.com/Security-Onion-Solutions/securityonion/discussions/10991.

I tried adding the $REDIRECTIT to the end of no_proxy_string, but given that during the initial install configuration it asks for the web access URL after configuring the proxy, it doesn't work.

So I just hardcoded my FQDN to the end of the string, but could a way be looked at into automatically adding it in?

Thanks for all the work!

**Comment by defensivedepth** [comment]
I've created an issue to track this: https://github.com/Security-Onion-Solutions/securityonion/issues/11024
